test/test_helper.rb

```diff
@@ -12,6 +12,14 @@ def open_test_file(name)
   File.new(File.join(File.dirname(__FILE__), name.split('.').last, name))
 end
 
+# kubeconfig files deviate from above convention.
+# They link to relaved certs etc. with various extensions, all in same dir.
+def config_file(name)
```

Comment:
can avoid global scope with:
```suggestion
MiniTest::Test.class_eval do
  def config_file(name)
```

Reply:
done